### ISSUE TITLE
Rewatch support

### ISSUE THREAD

**Issue by j2ghz** [razzeee/script.trakt#292]
Something like setting a date for each series, and when checking seen status, ignore scrobbles before that date

**Comment by thwaller** [comment]
Are you just looking to have the watched status cleared and to show as not watched? This would otherwise be recorded as a second watch. I do not develop on this plugin, but I am working through similar issues on my install.

Are you looking to change trakt.tv (or reporting to trakt.tv) or the watched status in Kodi itself?

**Comment by j2ghz** [comment]
Trakt supports multiple scrobbles on one episode, kodi has just watched/unwatched. Now this plugin converts it like `trakt watched one or more times? set in kodi as watched else set as unwatched` and I want `trakt watched one or more times after a specified date? set in kodi as watched else set as unwatched`.

> This would otherwise be recorded as a second watch.

I want it to be recorded on trakt as second watch.

> Are you just looking to have the watched status cleared and to show as not watched?

No, I want no change on trakt.tv

> Are you looking to change trakt.tv (or reporting to trakt.tv) or the watched status in Kodi itself?

Trakt.tv is okay as it is. Changes to Kodi would have to be big, I don't think they would do it. I mean changes to this addon.

**Comment by bachirica** [comment]
I join you in this feature request. It would be great to be able to control watched/unwatched episodes for a second/third time of an already watched show without losing the previous plays on trakt

**Comment by jeromewh** [comment]
So I see the feature was added... but how does it work?
I just tried to mark a show as unwatched but after the next sync all episodes were marked as watched again.
Am I doing something wrong?

**Comment by razzeee** [comment]
It was not added, so far. Not sure why you think that.

**Comment by vc4u** [comment]
This is old feature request, but wanted to know if it is something being worked on yet? :-)

**Comment by razzeee** [comment]
@vc4u no, feel free to go ahead and work on it. if you can find a sane design.

**Comment by glensc** [comment]
looks like kodi has single value for date, but counts plays:

```xml
        <playcount>2</playcount>
        <lastplayed>2016-02-27</lastplayed>
```

this is from export

**Comment by beschoenen** [comment]
Hey guys, I took a stab at implementing this feature. And initial impressions seem like it's working. I'm not making a PR yet as I'm not a 100% sure about the solution. But please leave some comments if you have time.

KODI does not update the TV Series menu after a sync, but it does show the correct status there after a restart (I'm still looking into that).
And keep in mind that this does delete your watch statuses from KODI's database!

Check out the code for yourself here: https://github.com/trakt/script.trakt/compare/main...beschoenen:script.trakt:feature/rewatch

**Comment by vnnegt** [comment]
> Hey guys, I took a stab at implementing this feature. And initial impressions seem like it's working. I'm not making a PR yet as I'm not a 100% sure about the solution. But please leave some comments if you have time.
> 
> KODI does not update the TV Series menu after a sync, but it does show the correct status there after a restart (I'm still looking into that). And keep in mind that this does delete your watch statuses from KODI's database!
> 
> Check out the code for yourself here: [main...beschoenen:script.trakt:feature/rewatch](https://github.com/trakt/script.trakt/compare/main...beschoenen:script.trakt:feature/rewatch)

I'm a year late, but I just found this after purchasing VIP and learning about the rewatch feature - can confirm that it's working for me as well

Thanks!
